guardian/parcel-react-typescript
# Update dependencies including emotion 11

## What does this change?

This updates all dependencies to the latest version.
This includes emotion 11 which requires some changes to imports and tsconfig changes to ensure the css prop still works.

## How to test

Clone the repo and check that npm install, npm start and npm run build all still work.

## How can we measure success?

The project compiles, builds and runs the local server.

## Have we considered potential risks?


## Images


## Accessibility
